app/javascript/listings/components/Modal.jsx

```diff
@@ -29,16 +29,15 @@ const Modal = ({
     </svg>
   );
 
+  // TODO: a11y needs to be fixed here. we currently have a button within a button.
+  // Also, why are we not using the crayons modal component and instead recreating it here?
   return (
-    <div
+    <dialog
```

Comment:
Going semantic using [\<dialog /\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog).

Reply:
Wow, that's neat!

Reply:
Yeah, this is going to be replaced by out `<Modal />` component in crayons,  so this will no longer be an issue @rhymes.

Reply:
@ludwiczakpawel, let's leave changing the `<Modal />` to the crayons component for now and just get this merged. I think you're right. Let's do that in another PR. @rhymes, I've reverted this to a `<div />` with `role=dialog` and `aria-modal="true"`.